src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -100,12 +103,12 @@ public void Dispose()
                     _textView.Closed -= OnTextViewClosed;
                 }
 
-                _owner = null;
+                _owner = null!;
                 _workspace = null;
                 _workspaceStatusService = null;
-                _registration = null;
-                _textView = null;
-                _subjectBuffer = null;
+                _registration = null!;
+                _textView = null!;
```

Comment:
Per our conversation on Teams, did we want to just stop doing this? Or have a bug tracking cleaning these up?

Reply:
Yes, I will file a separate tracking issue.

Reply:
https://github.com/dotnet/roslyn/issues/40166